### ISSUE TITLE
LibreOffice: Cannot open options dialog

### ISSUE THREAD

**Issue by danielnaber** [languagetool-org/languagetool#1353]
Using LibreOffice 6.0.2.1 and LT 4.4, I get: 

```
java.lang.NullPointerException
	at org.languagetool.gui.ConfigurationDialog.<init>(ConfigurationDialog.java:77)
	at org.languagetool.openoffice.ConfigThread.<init>(ConfigThread.java:51)
	at org.languagetool.openoffice.Main.runOptionsDialog(Main.java:201)
	at org.languagetool.openoffice.Main.trigger(Main.java:358)
```

@FredKruse Any idea? Deleting `~/.languagetool-ooo.cfg` doesn't help.

**Comment by danielnaber** [comment]
Also see https://forum.languagetool.org/t/problem-with-lt4-1/2935/28?u=dnaber

**Comment by danielnaber** [comment]
Fixed in 4.4.1
